src/components/HeaderWithLanguage/SimpleLanguageHeader.tsx

```diff
@@ -52,7 +52,7 @@ const SimpleLanguageHeader = ({
   );
 
   return (
-    <>
+    <div>
```

Comment:
Må vi?

Reply:
Må ikke, men isåfall burde margin under heading fjernes sånn at ikke spacing blir så stor ?
<img width="503" alt="Skjermbilde 2024-11-18 kl  10 20 16" src="https://github.com/user-attachments/assets/d5532efe-78a3-47b9-bd08-9d45dc575931">